### PR TITLE
Feature/project view

### PR DESCRIPTION
**Pull request by albert118**
Adds the project view screen

### CODE REVIEW

**Comment by albert118** [comment]
See commit: e04b3320023158f772ee01bf336dc435696980ff

File: src/frontend/src/pages/Groups/GroupsDataTable.tsx [hunk 1]
```diff
@@ -26,8 +34,16 @@ export function GroupsDataTable({ groups }) {
...
     return (
         <SmsDataTable rows={groups} headers={headers} className='groups-page__datatable'>
-            {({ rows, headers, getHeaderProps, getRowProps, getTableProps }) => (
+            {({ rows, headers, getHeaderProps, getRowProps, getTableProps, onInputChange }) => (
                 <TableContainer title='Groups'>
+                    <TableToolbar>
+                        <TableToolbarContent>
+                            <TableToolbarSearch onChange={onInputChange} />
+                            <Button onClick={() => navigate(`${AppRoutes.projects}/add`)}>
+                                Create A New Project
```
[the remark this answers is not in the document]
Comment (JaydenL33) [reply src/frontend/src/pages/Groups/GroupsDataTable.tsx:43]: This looks great!

File: src/frontend/src/pages/Projects/ProjectsDataTable.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+// @ts-nocheck
+
+// Most carbon components lack typescript definitions
+// this is notable around something complex like the DataTable used here,
+// so we disable ts for the whole file as a workaround this is exported back to to View though, which still enables TS.
```
Comment (JaydenL33) [inline src/frontend/src/pages/Projects/ProjectsDataTable.tsx:5]: oh god

**Comment by JaydenL33** [comment]
LGTM

**Review by JaydenL33 — APPROVED** [review]
LGTM

**Pull request merged**